AsyncDisplayKit/ASImageNode.mm

```diff
@@ -22,42 +23,34 @@
 
 @interface _ASImageNodeDrawParameters : NSObject
 
-@property (nonatomic, assign, readonly) BOOL cropEnabled;
 @property (nonatomic, assign) BOOL opaque;
-@property (nonatomic, retain) UIImage *image;
 @property (nonatomic, assign) CGRect bounds;
 @property (nonatomic, assign) CGFloat contentsScale;
 @property (nonatomic, retain) UIColor *backgroundColor;
 @property (nonatomic, assign) UIViewContentMode contentMode;
-@property (nonatomic, assign) CGRect cropRect;
-@property (nonatomic, copy) asimagenode_modification_block_t imageModificationBlock;
 
 @end
 
 // TODO: eliminate explicit parameters with a set of keys copied from the node
 @implementation _ASImageNodeDrawParameters
```

Comment:
If we end up keeping a drop parameters concept longer term, I wonder if we could use a struct for this and pass by reference as a void pointer or something? Definitely not a fully formed thought, just curious if we could further cut down on object cost for something called thousands of times in periods of a few seconds.  I will start by profiling this and may prove that it is unnecessary.